src/Catalyst.Benchmark/Catalyst.Abstractions/HashingBenchmark.cs

```diff
@@ -0,0 +1,104 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using Catalyst.Abstractions.Hashing;
+using Catalyst.Core.Modules.Hashing;
+using Catalyst.Protocol.Transaction;
+using Google.Protobuf;
+using Nethermind.Core.Extensions;
+using TheDotNetLeague.MultiFormats.MultiHash;
+
+namespace Catalyst.Benchmark.Catalyst.Core.Modules.Hashing
+{
+    [MemoryDiagnoser]
+    [SimpleJob(RuntimeMoniker.CoreRt30)]
+    public class HashingBenchmark
+    {
+        static HashingBenchmark()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Initialize all 'static fields' inline and remove the 'static constructor'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4569935)